frontend/src/services/http/http.service.ts

```diff
@@ -43,6 +43,26 @@ class Http {
       .catch(this.throwError);
   }
 
+  loadObject(url: string, options: Partial<HttpOptions> = {}): Promise<Blob> {
+    const {
+      method = HttpMethod.GET,
+      payload = null,
+      contentType,
+      hasAuth = true,
+      params,
+    } = options;
+    const headers = this.getHeaders(contentType, hasAuth);
+
+    return fetch(this.getUrl(url, params), {
```

Comment:
fetch in out app fill be in one place - http service

Reply:
and it is)
it is a fetch method for downloading files